src/server/server.ts

```diff
@@ -185,6 +190,27 @@ async function serveArticlePost(
 	}
 }
 
+async function serveEditionArticlePost(
+	req: Request,
+	res: ExpressResponse,
+	next: NextFunction,
+): Promise<void> {
+	try {
+		// req.body: should contain a direct response from capi
+		// fetched by the edition backend
+		const searchResponse = await capiSearchDecoder(req.body);
+		// this end point serve single article so take the first item only
+		const content = searchResponse.results[0];
+		const mockedRenderingRequest: RenderingRequest = {
+			content,
+		};
+		void serveArticle(mockedRenderingRequest, res, false);
+	} catch (e) {
+		console.error(e);
+		res.status(400).send('bad request');
```

Comment:
We use a logging package to avoid some performance problems with using `console` under high request load, and some custom middleware to handle errors:

```suggestion
		logger.error('This error occurred', e);
		next(e);
```